### DISCUSSION TITLE
Naabu 2.1.3 Windows binaries in repo are incorrect (still referring 2.1.2)

### DISCUSSION THREAD

**Discussion by roggenk** [projectdiscovery/naabu#609] [General]
The executable in [naabu_2.1.3_windows_amd64.zip] is still 2.1.2
(https://github.com/projectdiscovery/naabu/releases/download/v2.1.3/naabu_2.1.3_windows_amd64.zip)

Took me a while to figure out why -update was (not) working, including pdtm update mechanism.

**Comment by ehsandeep** [comment]
@roggenk should be fixed with - https://github.com/projectdiscovery/naabu/releases/tag/v2.1.4, thanks for flagging.
